examples/rcbd_two_var.R

```diff
@@ -0,0 +1,113 @@
+# This script runs a multivariate ANOVA based on one dependent continous
+# variable, two factors, and a factor that represents the block in a Random
+# Complete Block Design. It is based on the example given on page 36 of the
+# agricolae tutorial[1].
+#
+# To run the script with full output:
+#
+# source('rcbd_two_var.R', print.eval = TRUE)
+#
+# [1] https://cran.r-project.org/web/packages/agricolae/vignettes/tutorial.pdf
+#
+# NOTE : All lines shown to the user in the app are surrounded by this
+# separator: #-----------------------------#
+
+#-----------------------------------------------------------------------------#
+library('agricolae')  # for LSD.test()
+library('car')  # for levenTest()
+library('effects')
+#-----------------------------------------------------------------------------#
+
+sep <- function(n){
+  # This function simply prints a line of "=" to the screen as a separator and
+  # will not show up in the app. It is only here for nice printing.
+  line <- paste0(paste(replicate(n, "="), collapse = ""), '\n')
+  cat(line)
+}
+
+# Create some data. This represents an RCBD design with a factorial clone x
+# nitrogen with yield as the response variable.
+yield <- c(6, 7, 9, 13, 16, 20, 8, 8, 9,
+           7, 8, 8, 12, 17, 18, 10, 9, 12,
+           9, 9, 9, 14, 18, 21, 11, 12, 11,
+           8, 10, 10, 15, 16, 22, 9, 9, 9)
+block <- gl(4, 9)
+clone <- rep(gl(3, 3, labels = c('c1', 'c2', 'c3')), 4)
+nitrogen <- rep(gl(3, 1, labels = c('n1', 'n2', 'n3')), 12)
+my.data <- data.frame(block, clone, nitrogen, yield)
+write.csv(my.data, 'rcbd_two_var.csv', quote=FALSE, row.names=FALSE)
+
+# Load the data.
+#-----------------------------------------------------------------------------#
+my.data <- read.csv('rcbd_two_var.csv')
+my.data$block <- as.factor(my.data$block)
+#-----------------------------------------------------------------------------#
+
+# Construct the model.
+#-----------------------------------------------------------------------------#
+model <- aov(yield ~ block + clone + nitrogen + clone:nitrogen, data=my.data)
+#-----------------------------------------------------------------------------#
+
+# Plot the four standard fit plots: residuals vs predicted, sqrt of residuals vs
```

Comment:
I think we should remove these 2 plots: resid vs leverages and sqrt of residuals vs fitted